packages/frontend/stories/avatar.stories.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { text } from '@storybook/addon-knobs';
+
+import { storiesOf } from '@storybook/react';
+import Avatar from '../components/ui/avatar';
+
+storiesOf('Avatar', module)
+  .add('Small Avatar', () => (
+    <Avatar
+      className={text('Classname')}
+      type="xs"
+      image="/img/placeholder-image-02.jpg"
+    />
+  ))
+  .add('Normal Avatar', () => (
+    <Avatar
+      className={text('Classname')}
+      type="normal"
+      image="/img/placeholder-image-02.jpg"
+    />
+  ))
+  .add('Large Avatar', () => (
+    <Avatar
+      className={text('Classname')}
+      type="lg"
+      image="/img/placeholder-image-02.jpg"
+    />
+  ))
+  .add('Haven"t got Image Avatar', () => (
```

Comment:
`'Haven"t...'` isn't right. you can use it like `"Haven't..."` but lets change it with something else that explain itself more.

my suggestion: `'Avatar without Image'`